.github/ISSUE_TEMPLATE/task.md

```diff
@@ -1,7 +1,7 @@
 ---
 name: Task
 about: Describe something that needs to be done
-title: ''
+title: 'Internal: '
```

Comment:
I worry a bit about the use of `Internal` here in that there's potential for confusion in the release notes with issues coming from the METplus-Interal repository. Although using `title: 'Task: '` wouldn't make for a very meaningful organizational category in the release notes. But for consistency and simplicity, I would choose `Task` over `Internal`.
```suggestion
title: 'Task: '
```

Reply:
I agree about the confusion of including the "Internal: " prefix by default. A task issue could be a number of things, so it is not safe to assume what category it should go under. I removed the default title from the task template.